skbuild/src/harness.sk

```diff
@@ -248,36 +248,31 @@ fun buildLibrary(
   Some(target)
 }
 
-fun buildResource(env: Env, file: String): String {
+fun buildResource(env: Env, file: String, relToOut: Bool): String {
   checker = TimeChecker(mtime(file));
-  object = env.targetPath(file, ".o");
+  object = if (relToOut) mvExt(file, ".o") else env.targetPath(file, ".o");
   if (checker.isYoungerThan(object)) {
+    source = if (relToOut) mvExt(file, ".c") else env.targetPath(file, ".c");
     ensureDirectory(Path.dirname(object), env.verbose);
-    relative = env.relative(file);
-    nf = file.replace("/", "_").replace(".", "_");
-    nr = relative.replace("/", "_").replace(".", "_");
-    redefs = mutable Vector<String>[];
-    Array["start", "size", "end"].each(suffix -> {
-      redefs.push("--redefine-sym");
-      redefs.push(`_binary_${nf}_${suffix}=_binary_${nr}_${suffix}`);
-    });
-    (b, o) = copyObjOptions();
-    cmd = Array[
-      "objcopy",
-      "-B",
-      b,
-      "-I",
-      "binary",
-      "-O",
-      o,
-      file,
-      object,
```

Comment:
Question: Is the change from calling objcopy to using the `copyBinary` function only because macos does not ship with gnu objcopy, or is there another motivation? I wonder if since we will need llvm to be installed anyhow, maybe changing this use of `objcopy` to `llvm-objcopy` would be enough?

Reply:
That's exactly the motivation, and indeed there is llvm-objcopy, however I have not been able to get it to work correctly on macos. This is the simplest solution I've found to handle all platforms.